Humin-Man/Startup.cs

```diff
@@ -61,7 +61,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
-                app.UseDatabaseErrorPage();
+                app.UseMigrationsEndPoint();
```

Comment:
Why did you change this?

Reply:
i was getting an error and found out that `app.UseDatabaseErrorPage()` is obsolete 
https://github.com/aspnet/Announcements/issues/432